pkg/configuration/configuration.go

```diff
@@ -28,6 +38,7 @@ type Config struct {
 	DeviceID    string  `mapstructure:"device_id"`
 	PemFilePath string  `mapstructure:"pem_filepath"`
 	Timeout     int     // HTTP timeout, not supported in the config file.
+	Meta        Meta    `mapstructure:"meta"`
```

Comment:
I think we should keep a flat config structure. Right now there is no way to disable version check through environment variables. but If we would use a flat config, viper would take care of it for us.

Reply:
Good point. I'll fix that as well